gem2rpm/foreman_plugin.spec.erb

```diff
@@ -101,15 +107,11 @@ Documentation for %{name}.
 <% end # if doc_subpackage -%>
 
 %prep
-gem unpack %{SOURCE0}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%setup -q -n  %{gem_name}-%{version}
 
 %build
 # Create the gem as gem install only works on a gem file
-gem build %{gem_name}.gemspec
+gem build ../%{gem_name}-%{version}.gemspec
```

Comment:
Curious about this change, why is the location at a different level for plugins?

Reply:
This is aligning it to `default.spec.erb` where we made the same change.

Reply:
Oh right, the `../` threw me.